src/Symfony/Component/RateLimiter/Policy/NoLimiter.php

```diff
@@ -29,12 +29,12 @@ final class NoLimiter implements LimiterInterface
 {
     public function reserve(int $tokens = 1, ?float $maxTime = null): Reservation
     {
-        return new Reservation(time(), new RateLimit(\INF, new \DateTimeImmutable(), true, \INF));
```

Comment:
I am not into the code nor the component, but couldn't we use "-1" like for example done in memory_limit setting in PHP instead of a very large number? 🧐

cc @wouterj

Reply:
Currently the only use of the remainingTokens and limit properties is for the developer to know how many requests the user can send. This is not used internally in the RateLimiter component.